Composer/packages/client/src/components/ImportModal/ImportModal.tsx

```diff
@@ -0,0 +1,338 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { navigate, RouteComponentProps } from '@reach/router';
+import { Dialog, DialogType, DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
```

Comment:
Have you considered using DialogWrapper from shared ui lib? It basically uses the same Dialog from Fabric but allows us to do app wide customization+styling in the future.

Reply:
I don't really see the benefit in defining a new Dialog type over in the DialogWrapper for the styles used in the Import flow.

I would have to extend the DialogWrapper props to allow me to control the `isBlocking` and whether or not the close button shows on the dialog.

The dialog styles I'm using in this flow are only shared between different states of `ImportStatus`, so if I need to change the styles, I can just change the styles object in that file.

If this becomes a problem in the future, I can change it as needed.